components/style/properties/properties.mako.rs

```diff
@@ -3275,14 +3275,6 @@ where
         rule_cache_conditions: RefCell::new(rule_cache_conditions),
     };
 
-    let ignore_colors = !device.use_document_colors();
```

Comment:
Why are you removing this? It doesn't seem to me invoking `use_document_colors` is as cheap as checking a variable. Maybe optimizer can extract that as a loop invariant, but this seems to be unrelated to this commit anyway.

Reply:
We only use it once now.

Reply:
Why? It seems to me we are still checking this for every declaration.

Reply:
Oh, right... whoops. Will do that